adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -162,6 +162,10 @@ func CreateRoutesWithClusters(mgwSwagger model.MgwSwagger, upstreamCerts []byte,
 			logger.LoggerOasparser.Warnf("Production environment endpoints are not available for the resource %v:%v-%v",
 				apiTitle, apiVersion, resource.GetPath())
 		}
+		// Check whether the endpoint contains a trailing slash.
+		if string(endpointBasepath) == "/" {
```

Comment:
Should not we generalize such that we check for the trailing `/`

Reply:
Fixed via https://github.com/wso2/product-microgateway/pull/2167/commits/d3449286f728bf04395dd35a9c6701ce19c3a36a